src/widgets/Menu/Accordion.tsx

```diff
@@ -38,8 +40,8 @@ const Accordion: React.FC<Props> = ({ label, icon, isPushed, pushNav, initialOpe
   };
 
   return (
-    <Container onClick={handleClick}>
-      <MenuEntry>
+    <Container>
+      <MenuEntry onClick={handleClick}>
         {icon}
```

Comment:
Now that we have relative links we need to keep the accordion open when children are clicked.